public/apps/configuration/app-router.tsx

```diff
@@ -232,7 +241,14 @@ export function AppRouter(props: AppDependencies) {
               path={ROUTE_MAP.tenants.href}
               render={() => {
                 setGlobalBreadcrumbs(ResourceType.tenants);
-                return <TenantList {...props} />;
+                return <TenantList tabID={'Manage'} {...props} />;
```

Comment:
How will i18n work here?

Reply:
You can look at usages of `translateMessage` for how to do i18n in this repo: https://github.com/opensearch-project/security-dashboards-plugin/blob/main/public/apps/translation-utils.ts#L18-L20

Reply:
@cwperks We need to pass tabID in order to switch tabs.

Reply:
It can not be a constant as we want to call TenantList with different tabs based on our use case.